lib/shared/permissions.js

```diff
@@ -119,6 +144,45 @@ exports.getEnvironmentCommandPrefix = (environment) => {
 };
 
 /**
+ * @summary Build the elevate command
+ * @function
+ * @private
+ *
+ * @param {String[]} command - command and arguments
+ * @param {Object} [environment] - environment variables
+ * @returns {String[]} elevate command
+ *
+ * @example
+ * const command = permissions.buildElevateCommand([ 'foo', 'bar' ], {
+ *   ELECTRON_RUN_AS_NODE: 1
+ * });
+ *
+ * console.log(command);
+ *
+ * // Windows
+ * [ 'cmd.exe', '/c', '"set "ELECTRON_RUN_AS_NODE=1" && call foo bar"' ]
+ *
+ * // Unix
+ * [ 'env', 'ELECTRON_RUN_AS_NODE="1"', 'foo', 'bar' ]
+ */
+exports.buildElevateCommand = (command, environment = {}) => {
+  const prefixedCommand = _.concat(
+    exports.getEnvironmentCommandPrefix(environment),
+    _.map(command, quoteAndEscapeString)
+  );
+
+  if (os.platform() === 'win32') {
```

Comment:
Given how many times we do this check, would it be worth doing a single `const isWindows = os.platform() === 'win32';` at the module-level scope? I think it's unlikely that the OS platform will change during the execution-lifetime of the script ;-)